desktop/src/com/mygdx/game/client/network/ComponentMessageListener.java

```diff
@@ -20,39 +23,49 @@
 @Log
 public class ComponentMessageListener extends AbstractWebSocketListener {
 
-  /**
-   * Used as default value when invoking {@link ObjectMap#get(Object, Object)} on {@link #handlers} to prevent
-   * NPE.
-   */
-  private static final Handler<Component> unknown = (webSocket, packet) -> NOT_HANDLED;
-  private final ObjectMap<Class<?>, Handler<Component>> handlers = new ObjectMap<>();
-  private final Json json = new Json();
-  private final ComponentMapper<Position> positionMapper;
+  private static final Handler NO_HANDLER = (webSocket, worldEntity, packet) -> {
+    throw new RuntimeException("Every ComponentMessage needs to be handled: id=" + worldEntity
+        + " content=" + packet);
+  };
+  private final ObjectMap<Class<? extends Component>, Handler> handlers = new ObjectMap<>();
```

Comment:
Ta klasa dobrze handluje teraz ComponentMessage<jakiś komponent>, nadużyłem ją jeszcze do handlowania EntityConfigId, ale to nie musi być tak robione, może być jako osobna wiadomość.